spec/requests/reservations_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe "Reservations", type: :model do
```

Comment:
describe メソッドの第 1 引数にはテスト対象のクラスを指定するのが一般的です。

```suggestion
RSpec.describe Reservation, type: :model do
```

Reply:
こちらで対応しました！
https://github.com/nisimotoryoga/pbl_fp_matching/pull/57/commits/aa066811a483580a6fbb9f1876fb4bdf317f6e5f